cmake/protobuf-options.cmake

```diff
@@ -0,0 +1,7 @@
+# Verbose output

```

Comment:
It's a trick :-) I include these file in the main CMakeListst.txt file, and it is used in protobuf-config.cmake.in for all users after installation from package configuration.
This allows these options determine only once.